lib/elixir/test/elixir/kernel/errors_test.exs

```diff
@@ -818,21 +818,18 @@ defmodule Kernel.ErrorsTest do
     end
   end
 
-  test "Kernel.def fails when rescue, else or catch don't have clauses" do
+  test "def fails when rescue, else or catch don't have clauses" do
     assert_eval_raise CompileError,
       ~r"expected -> clauses for :else in \"def\"",
       """
-      defmodule A do
-        def hi do
-          e()
+      defmodule Example do
+        def foo do
+          bar()
         else
-          x()
+          baz()
         end
       end
       """
-  after
-    :code.purge A
-    :code.delete A
```

Comment:
@lexmag, I did it because @josevalim asked me not to forget to purge the created module.

Reply:
Hmm, `:code.is_loaded(A)` gives me `false`.

Reply:
I said it by mistake because I thought it was a warning and not an error. Thanks for fixing it!